compiler/src/dotty/tools/dotc/core/Definitions.scala

```diff
@@ -585,6 +585,10 @@ class Definitions {
 
   lazy val PartialFunctionType: TypeRef         = ctx.requiredClassRef("scala.PartialFunction")
   def PartialFunctionClass(implicit ctx: Context) = PartialFunctionType.symbol.asClass
+
+    lazy val PartialFunction_applyOrElseR = PartialFunctionClass.requiredMethodRef(nme.applyOrElse)
```

Comment:
This has the wrong indent size?

Reply:
This is the indentation scheme used across the file